### PR TITLE
2.28.0

### PR DESCRIPTION
**Pull request by Issif**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

> /area config
/area outputs

> /area tests

**What this PR does / why we need it**:

release 2.28.0

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,39 +1,75 @@
 # Changelog
 
+## 2.28.0 - 2023-07-18
+#### New
+- New output: **Redis** ([PR#396](https://github.com/falcosecurity/falcosidekick/pull/396) thanks to [@pandyamarut](https://github.com/pandyamarut))
+- New output: **Telegram** ([PR#431](https://github.com/falcosecurity/falcosidekick/pull/431) thanks to [@zufardhiyaulhaq](https://github.com/zufardhiyaulhaq))
+- New output: **Grafana OnCall** ([PR#470](https://github.com/falcosecurity/falcosidekick/pull/470))
+- New output: **OpenObserve** ([PR#509](https://github.com/falcosecurity/falcosidekick/pull/509))
+
```
Comment (Lowaiz) [inline CHANGELOG.md:9]:
```suggestion
- New output: **N8N** ([PR#509](https://github.com/falcosecurity/falcosidekick/pull/462))
```

Comment (Issif) [reply CHANGELOG.md:10]: Fixed. Thank you

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,39 +1,76 @@
 # Changelog
 
+## 2.28.0 - 2023-07-18
+#### New
+- New output: **Redis** ([PR#396](https://github.com/falcosecurity/falcosidekick/pull/396) thanks to [@pandyamarut](https://github.com/pandyamarut))
+- New output: **Telegram** ([PR#431](https://github.com/falcosecurity/falcosidekick/pull/431) thanks to [@zufardhiyaulhaq](https://github.com/zufardhiyaulhaq))
+- New output: **N8N** ([PR#509](https://github.com/falcosecurity/falcosidekick/pull/462))
```
Comment (Lowaiz) [inline CHANGELOG.md:7]:
```suggestion
- New output: **N8N** ([PR#462](https://github.com/falcosecurity/falcosidekick/pull/462))
```

Comment (Lowaiz) [reply CHANGELOG.md:7]: I copye pasted a bit to fast :D

**Review by cpanato — APPROVED** [review]
/lgtm

thank you

**Pull request merged**